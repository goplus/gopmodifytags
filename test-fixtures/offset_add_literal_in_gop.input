var in struct {
	Foo string
}
