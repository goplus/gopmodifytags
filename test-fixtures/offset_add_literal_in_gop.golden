var in struct {
	Foo string `json:"foo"`
}
